include/sec_security.h

```diff
@@ -16,8 +16,8 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#ifndef SEC_SECURITY_H
-#define SEC_SECURITY_H
+#ifndef SEC_SECURITY_H_
```

Comment:
Why is the succeeding underscore needed for the file guard? This doesn't match our precedent in other code bases.